src/mockData.js

```diff
@@ -0,0 +1,106 @@
+//user_id might only be so helpful, we might want user name instead?
+export const allRecipes = [
+  {
+    id: 1,
+    user_id: 1,
+    image: "https://www.chasinglenscapes.com/wp-content/uploads/2020/06/food-photography-on-the-go-tips.jpg",
+    title: 'Egg Stuff',
+    description: "It looks like bread... and eggs? I think you want it. Donate to make it.",
+    instructions: "1. Crack and egg. 2. Cook it. 3. Then do more stuff. 4. Then even more.",
```

Comment:
This is how I describe my own recipes. 10/10. Thank you for taking the time to mock out the data!